### PR TITLE
Initial commit of digital ocean guide

### PR DESCRIPTION
**Pull request by lucywyman**
@ElijahCaine @rettigs @tschuy

### CODE REVIEW

File: content/guides/digital_ocean.mkd [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   you. 
+4. Go to [Digital Ocean][do] and sign up for an account, using the
+   key provided by github in the Coupon section
+5. Now you can make a droplet! Login to Digital Ocean, and click the
+   green "Create a Droplet" button in the top-right-hand corner.
+6. If you're not sure what you want, name your droplet something 
+   sensible ("Schoolwork", "Learning Linux", "IRC", etc.), then choose
+   the $5/month sized machine -- this will more than likely be 
+   more than enough space for right now.
+7. Select either the San Francisco or New York Regions, and the
```
Comment (tschuy) [inline content/guides/digital_ocean.mkd]: I would not recommend the New York region, I have a droplet there and running things on it is annoying. I've heard SF is better. 

(I have no idea if it's actually just the $5 VMs being that bad, or if it's a case of annoying ping. Probably a bad combo of the two.)

**Comment by edunham** [comment]
Were you able to successfully run wok to build the site with these changes? I think you need to either specify a slug for the page or or set `slug_from_filename: True` in the site's config.

**Comment by lucywyman** [comment]
Fuck me

**Comment by lucywyman** [comment]
Closed in favor of #70

**Pull request closed without merging**